### PR TITLE
Feature/hl poi reachability readme

### PR DESCRIPTION
**Pull request by HLucasTargomo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -317,6 +320,27 @@ Return possible geocode(s) for each given address.
...
 
+## Poi Reachability service
+
+Return reachable POIs from a list of sources
+
+    TravelOptions options = new TravelOptions();
+    options.setMaxEdgeWeight(900);
+    options.setEdgeWeightType(EdgeWeightType.TIME);
+    options.addSource(new DefaultSourceCoordinate("id1", 40.608155, -73.976636));
+    options.setOsmTypes(Collections.singleton(new PoiType("amenity", "restaurant")));
```
Comment (DavidRedlich) [inline README.md:331]: can you leave a comment anywhere that this refers to Open Street Map poi types (with link to the different types)

maybe not here but maybe in the code somewhere?

**Review by atomicbomberman — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (HLucasTargomo) [reply README.md:331]: with a link to osm wiki?

Comment (DavidRedlich) [reply README.md:331]: yeah, maybe? otherwiese people might not be able to define appropriate "key" "value" pairs to use it

Comment (HLucasTargomo) [reply README.md:331]: I added a link :)

**Review by DavidRedlich — APPROVED** [review]

**Pull request merged**